app/vlinsert/insertutil/common_params.go

```diff
@@ -326,10 +332,13 @@ func (cp *CommonParams) NewLogMessageProcessor(protocolName string, isStreamMode
 		lmp.initPeriodicFlush()
 	}
 
+	messageProcessorCount.Add(1)
 	return lmp
 }
 
 var (
 	rowsDroppedTotalDebug         = metrics.NewCounter(`vl_rows_dropped_total{reason="debug"}`)
 	rowsDroppedTotalTooManyFields = metrics.NewCounter(`vl_rows_dropped_total{reason="too_many_fields"}`)
+	_                             = metrics.NewGauge(`vl_insert_processors_count{type=""}`, func() float64 { return float64(messageProcessorCount.Load()) })
```

Comment:
The `{type=""}` isn't needed here, since empty labels are ignored by VictoriaMetrics and Prometheus during scrape.